pkg/schedule/scatter/region_scatterer.go

```diff
@@ -272,6 +276,25 @@ func (r *RegionScatterer) Scatter(region *core.RegionInfo, group string, skipSto
 		return nil, errors.Errorf("region %d is not fully replicated", region.GetID())
 	}
 
+	if op := r.opController.GetOperator(region.GetID()); op != nil {
+		val, ok := op.AdditionalInfos["group"]
+		// If the existing operator is created by the same group scatterer, just skip creating a new one.
+		if strings.Contains(op.Desc(), scatterOperatorDes) && ok && val == group {
```

Comment:
They are not in 10022?

Reply:
Yes，but this hotfix needs it to ensure not create too many operator in retry